packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -30,6 +30,9 @@ export interface IInputGroupProps extends IControlledProps, IIntentProps, IProps
      */
     leftIcon?: IconName | JSX.Element;
 
+    /** Change event handler. Use `event.target.value` for new value. */
+    onChange: React.FormEventHandler<HTMLInputElement>
```

Comment:
_definitely_ gonna want this optional. also it likely conflicts with React's own HTML prop types....